<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keyboardDlg</class>
 <widget class="QWidget" name="keyboardDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="num0">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>14</y>
     <width>74</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="charMinus">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>71</y>
     <width>74</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EXP">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>71</y>
     <width>74</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>EXP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Enter">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>71</y>
     <width>81</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>14</y>
     <width>74</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num4">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>14</y>
     <width>74</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num5">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>14</y>
     <width>74</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num7">
   <property name="geometry">
    <rect>
     <x>558</x>
     <y>14</y>
     <width>74</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num6">
   <property name="geometry">
    <rect>
     <x>478</x>
     <y>14</y>
     <width>74</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num8">
   <property name="geometry">
    <rect>
     <x>637</x>
     <y>14</y>
     <width>74</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="charDot">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>71</y>
     <width>74</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>488</x>
     <y>71</y>
     <width>81</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cleanInput">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>71</y>
     <width>81</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clean</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num9">
   <property name="geometry">
    <rect>
     <x>717</x>
     <y>14</y>
     <width>74</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>14</y>
     <width>74</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num1">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>14</y>
     <width>74</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="showInput">
   <property name="geometry">
    <rect>
     <x>667</x>
     <y>71</y>
     <width>121</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="plus">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>74</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="percent">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>71</y>
     <width>74</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>105</y>
     <width>791</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>20</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
